checker/src/main/java/org/checkerframework/checker/initialization/InitializationStore.java

```diff
@@ -59,72 +52,30 @@ public InitializationStore(CFAbstractAnalysis<V, S, ?> analysis, boolean sequent
      * initialized.
      */
     @Override
-    public void insertValue(JavaExpression je, V value, boolean permitNondeterministic) {
+    public void insertValue(JavaExpression je, CFValue value, boolean permitNondeterministic) {
         if (!shouldInsert(je, value, permitNondeterministic)) {
             return;
         }
 
-        InitializationAnnotatedTypeFactory<?, ?, ?, ?> atypeFactory =
-                (InitializationAnnotatedTypeFactory<?, ?, ?, ?>) analysis.getTypeFactory();
-        QualifierHierarchy qualifierHierarchy = atypeFactory.getQualifierHierarchy();
-        AnnotationMirror invariantAnno = atypeFactory.getFieldInvariantAnnotation();
-
-        // Remember fields that have the 'invariant' annotation in the store.
-        if (je instanceof FieldAccess) {
-            FieldAccess fieldAccess = (FieldAccess) je;
-            if (!fieldValues.containsKey(je)) {
-                AnnotationMirrorSet declaredAnnos =
-                        atypeFactory.getAnnotatedType(fieldAccess.getField()).getAnnotations();
-                if (AnnotationUtils.containsSame(declaredAnnos, invariantAnno)) {
-                    if (!invariantFields.containsKey(fieldAccess)) {
-                        invariantFields.put(
-                                fieldAccess,
-                                analysis.createSingleAnnotationValue(invariantAnno, je.getType()));
-                    }
-                }
-            }
-        }
-
         super.insertValue(je, value, permitNondeterministic);
 
-        for (AnnotationMirror a : value.getAnnotations()) {
-            if (qualifierHierarchy.isSubtype(a, invariantAnno)) {
-                if (je instanceof FieldAccess) {
-                    FieldAccess fa = (FieldAccess) je;
-                    if (fa.getReceiver() instanceof ThisReference
-                            || fa.getReceiver() instanceof ClassName) {
-                        addInitializedField(fa.getField());
-                    }
-                }
+        if (je instanceof FieldAccess) {
+            FieldAccess fa = (FieldAccess) je;
+            if (fa.getReceiver() instanceof ThisReference
+                    || fa.getReceiver() instanceof ClassName) {
+                addInitializedField(fa.getField());
             }
         }
     }
 
     /**
-     * {@inheritDoc}
+     * A copy constructor.
      *
-     * <p>Additionally, the {@link InitializationStore} keeps all field values for fields that have
-     * the 'invariant' annotation.
+     * @param other the store to copy.
```

Comment:
```suggestion
     * @param other the store to copy
```

Throughout, make sure to not end `@param` comments with a period.